vpr/src/analytical_place/ap_mass_report.cpp

```diff
@@ -0,0 +1,542 @@
+/**
+ * @file
+ * @author  Alex Singer
+ * @date    May 2025
+ * @brief   Implementation of the AP mass report generator.
+ */
+
+#include "ap_mass_report.h"
+#include <cmath>
+#include <fstream>
+#include <iomanip>
+#include <ostream>
+#include <string>
+#include <vector>
+#include "ap_netlist.h"
+#include "ap_netlist_fwd.h"
+#include "globals.h"
+#include "logic_types.h"
+#include "physical_types.h"
+#include "physical_types_util.h"
+#include "primitive_vector.h"
+#include "vpr_context.h"
+#include "vpr_error.h"
+#include "vtr_time.h"
+#include "vtr_vector.h"
+
+namespace {
```

Comment:
Just out of curiosity: I think the main reason for using an anonymous namespace here is because of the structs. But if the structs weren't present, is there any real difference between using static functions and defining functions in an anonymous namespace?